includes/image-block.html

```diff
@@ -0,0 +1,5 @@
+{% assign imageBlockImage = include.imageBlock %}
+
+<image-block
+  :image="{{ imageBlockImage | jsonify | escape }}"
```

Comment:
Müsste das nicht :image-block sein und imageBlock anstatt imageBlockImage? So hast du dann auch nicht in der Komponente sowas weirdes this.image['image']